dcos/dcos-ui
# DCOS-62249: Ph/feat/vertical bursting service form

CLOSES DCOS-62249
## Testing

To test this PR:
Settings will not yet be persisted. 
1) visit the service form
1.1) single container
  - expend more settings in the general section
  - Find limits `cpus` and `memory` fields
  - enter values
  - open JSON editor and check if the correct values are set in the JSON.
1.2) Multi Container
  - Visit the `container-1` section
  - expend more settings in the general section
  - Find limits `cpus` and `memory` fields
  - enter values
  - open JSON editor and check if the correct values are set in the JSON.

## Trade-offs



## Dependencies



## Screenshots
